velox/exec/PartitionedOutput.h

```diff
@@ -23,6 +23,8 @@ namespace facebook::velox::exec {
 
 class PartitionedOutput;
 
+// Base class for both streaming and non-streaming Destinations
```

Comment:
The logic using targetSizePct_, targetNumRows_, rng_ member variables seems to be specific to StreamingDestination. Should we move these variables out of the base class?

Reply:
Great catch! I'll fix that.